git-cliff-core/src/repo.rs

```diff
@@ -466,7 +466,15 @@ impl Repository {
             }
         }
         if !topo_order {
-            tags.sort_by(|a, b| a.0.time().seconds().cmp(&b.0.time().seconds()));
+            tags.sort_by(|a, b| {
+                let time_cmp = a.0.time().seconds().cmp(&b.0.time().seconds());
+                if time_cmp == std::cmp::Ordering::Equal {
+                    // When commit times are equal, sort by semantic version
+                    semantic_version_compare(&a.1.name, &b.1.name)
```

Comment:
I'm not sure if I understood how this fixes #1080... In that issue we didn't talk about commit times being equal etc. but we are doing a a check here.

Can you explain it further?